prime-router/src/main/kotlin/Report.kt

```diff
@@ -222,7 +222,7 @@ class Report {
         }
 
         fun formFileName(id: ReportId, schemaName: String, fileFormat: OrganizationService.Format?, createdDateTime: OffsetDateTime): String {
-            val formatter = DateTimeFormatter.ofPattern("YYYYMMDDhhmmss")
+            val formatter = DateTimeFormatter.ofPattern("YYYYMMddHHmmss")
```

Comment:
"DD" is day of year - changed to dd, day of month.
hh is 12 hour day - changed to HH, 24 hour day.